Week3/homework/step4-bonus.js

```diff
@@ -1,10 +1,20 @@
 'use strict';
 
-const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
-
+//const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
+const values = [1, 2, 3, 4, 5, 2, 3, 5, 7, 8, 9];
 // Add your function here. Try and come up with a good name for this function
-
+function findUniqElement(arr) {
+  const newValues = [];
+  while (arr.length > 0) {
+    if (newValues.includes(arr[0]))
+      arr.shift();
```

Comment:
Use curly braces around the `if` and `else` blocks; see my previous comment about this.